README.md

```diff
@@ -2,84 +2,80 @@
+And many other questions from many students trying to make their study plan for the semester. And we are one of them.
 
-การวางแผนลงทะเบียนในแต่ละเทอมเมื่อก่อนยุ่งยากและเสียเวลามาก ๆ ข้อมูลวิชาเรียนกระจัดกระจาย โดยเฉพาะอย่างยิ่งวิชา GenEd การจะหาวิชา GenEd ที่ใช่ เวลาเรียนที่ลงตัว ต้องหาข้อมูลจากหลากหลายแหล่งเพื่อประกอบการตัดสินใจ อีกทั้งยังมีเรื่องที่นั่งและความนิยมของวิชา GenEd เป็นปัจจัยสำคัญต่อการตัดสินใจ แต่ข้อมูลเหล่านั้นกลับเข้าถึงได้ยาก ทำให้วางแผนได้ยากลำบาก
+Picking courses and sections in each semester is a very complicated and time-consuming process. The course information are scattered, espescially GenEd courses. When trying to find the right GenEd, you have to consider many factors such as course contents, available seats, and popularity. But those information are hard to access, which makes planning very hard.
 
-[ทีมผู้พัฒนา](#developers) เห็นถึงปัญหานี้ จึงได้วางแผนจัดทำเว็บไซต์เพื่ออำนวยความสะดวกในการวางแผนลงทะเบียนขึ้นมา
+We saw this problem and built [CU Get Reg](https://cugetreg.com) to make this planning process a breeze.
 
 <div id="related-repositories"></div>
 
-## โปรเจคที่เกี่ยวข้อง
+## Related repositories
 
-- [cugetreg-frontend](https://github.com/thinc-org/cugetreg-frontend) เว็บไซต์ CU Get Reg
-- [cugetreg-backend](https://github.com/thinc-org/cugetreg-backend) ระบบหลังบ้านของเว็บไซต์ CU Get Reg
-- [cugetreg-scraper](https://github.com/thinc-org/cugetreg-scraper) ระบบคัดลอกข้อมูลวิชาเรียนแบบสาธารณะจาก [Reg Chula](https://cas.reg.chula.ac.th/cu/cs/QueryCourseScheduleNew/index.html)
-- [chula-courses](https://github.com/thinc-org/chula-courses) Schema ส่วนกลางของข้อมูลรายวิชา
+- [cugetreg-frontend](https://github.com/thinc-org/cugetreg-frontend) The web interface that users see
+- [cugetreg-backend](https://github.com/thinc-org/cugetreg-backend) The system that powers this site
+- [cugetreg-scraper](https://github.com/thinc-org/cugetreg-scraper) The system that fetches public course data from [Reg Chula](https://cas.reg.chula.ac.th/cu/cs/QueryCourseScheduleNew/index.html)
+- [chula-courses](https://github.com/thinc-org/chula-courses) Data schema
 
 <div id="contribution"></div>
 
-## มาเป็นส่วนหนึ่งกับเรา
+## Contribution
 
-พวกเรา [ทีมผู้พัฒนา](#developers) ยินดีต้อนรับนักพัฒนาทุกคน! มาร่วมเป็นส่วนหนึ่งในการพัฒนาสังคมของจุฬาลงกรณ์วิทยาลัยให้ดียิ่ง ๆ ขึ้นไปอีก! หากคุณส่วนรู้สึกสนใจที่จะมีส่วนร่วมกับเราแล้ว คุณสามารถเริ่มได้ที่นี่ได้เลย
+We welcome all developers. If you want to be a part of CU Get Reg and improve Chulalongkorn University society, you can start here!
 
 - [Contribution Guides](#https://github.com/thinc-org/cugetreg-frontend/wiki/Contribution-Guides)
 - [Set up your local environments](https://github.com/thinc-org/cugetreg-frontend/wiki/Contribution-Guides)
 - [Explore issues](https://github.com/thinc-org/cugetreg-frontend/issues)
 - [Our Roadmap](https://github.com/thinc-org/cugetreg-frontend/wiki/Contribution-Guides)
 
-หากคุณยังรู้สึกลังเล คุณสามารถ [ติดต่อเรา](#contacts) เพื่อปรึกษาก่อนได้นะ
```

Comment:
not sure about this, feel free to ignore it if you think it's weird
```md
If you have any questions, feel free to [contact us](#contacts) to discuss potential contributions.
```

Reply:
Idk about this, it feels weird